nltk/translate/meteor_score.py

```diff
@@ -8,27 +8,38 @@
 
 
 from itertools import chain, product
+from typing import Iterable
```

Comment:
Does this get used? I'm personally down to use the Python 3.5+ typing in signatures, and use e.g.
```python
def _generate_enums(
    hypothesis: Iterable[str],
    reference: Iterable[str],
    preprocess: Callable[[str], str] = str.lower
) -> Tuple[List[Tuple[str, str]], List[Tuple[str, str]]]:
    """...
    
    :param hypothesis: pre-tokenized hypothesis
    :type hypothesis: Iterable[str]
    :param reference: pre-tokenized reference
    :type reference: Iterable[str]
    :preprocess: preprocessing method (default str.lower)
    :type preprocess: Callable[[str], str]
    :return: enumerated words list
    :rtype: list of 2D tuples, list of 2D tuples
    :rtype: Tuple[List[Tuple[str, str]], List[Tuple[str, str]]]
    """
    ...
```
*However*, this isn't necessarily in the scope of your PR. This is a change I'd like to see throughout the entire codebase, so you can just leave it like it was if you prefer. At that point it might also be overkill to specify the type *both* in the signature and in the docstring.